src/Compilers/CSharp/Portable/FlowAnalysis/DefiniteAssignment.cs

```diff
@@ -81,6 +81,13 @@ internal partial class DefiniteAssignmentPass : LocalDataFlowPass<
         /// </summary>
         private readonly PooledHashSet<Symbol> _writtenVariables = PooledHashSet<Symbol>.GetInstance();
 
+        /// <summary>
+        /// Struct fields that are implicitly initialized, due to being used before being written, or not being written at an exit point.
+        /// </summary>
+        private PooledHashSet<FieldSymbol>? _implicitlyInitializedFieldsOpt;
+
+        private PooledHashSet<FieldSymbol> ImplicitlyInitializedFields => _implicitlyInitializedFieldsOpt ??= PooledHashSet<FieldSymbol>.GetInstance();
```

Comment:
>ImplicitlyInitializedFields [](http://example.com/codeflow?start=43&length=27)

Minor: Consider replacing with an `AddImplicitlyInitializedField(FieldSymbol)` method to avoid use of this property when the field should be used instead.